tests/integration/standard/test_connection.py

```diff
@@ -398,3 +398,49 @@ def setUp(self):
             raise unittest.SkipTest(
                 'libev does not appear to be installed properly')
         ConnectionTests.setUp(self)
+
+    def test_watchers_are_finished(self):
```

Comment:
Does this need to be decorated to only run with the `libev` reactor or anything like that?